src/printer/tir_text_printer.cc

```diff
@@ -606,6 +606,27 @@ Doc TIRTextPrinter::VisitStmt_(const PrefetchNode* op) {
   return doc;
 }
 
+Doc TIRTextPrinter::VisitStmt_(const SparseIterationNode* op) {
```

Comment:
`TIRTextPrinter` was deprecated in TVM mainline but thanks for doing this as we haven't upstreamed yet.

Reply:
Yes we found it difficult to integrate into other systems as a fork of TVM and merging to mainline sounds like the best choice.

Reply:
Totally agree with you! Since other parts of this repo are outdated,only if merge this to mainline,can this module be truly used by the community.